integration/mocha.opts

```diff
@@ -3,4 +3,4 @@
 --require mock-local-storage
 --full-trace
 --exit
---timeout 300000
+--timeout 500000
```

Comment:
did this help? or should be go back to 300000?

Reply:
changing it back